processor.js

```diff
@@ -98,3 +126,26 @@ async function getRepoConfig(context) {
 
   return parsePackageJson(pullieRcRes.data);
 }
+
+/**
+ * Get org-level config for the org specified in the context
+ *
+ * @param {ProbotContext} context PR webhook context
+ * @returns {Promise<Object>} Config for the repo
+ */
+async function getOrgConfig(context) {
+  let pullieRcRes = {};
+  try {
+    pullieRcRes = await context.github.repos.getContents({
+      owner: context.payload.repository.owner.login,
+      repo: '.github',
+      path: '.pullierc'
+    });
+  } catch (err) {
+    // If there's no .pullierc, just skip this request. Otherwise, re-throw the error.
+    if (err.status === 404) return;
+    throw err;
+  }
+
+  return parsePackageJson(pullieRcRes.data);
```

Comment:
I would rename `parsePackageJson` to `parseBase64Json` since `pullieRcRes.data` is clearly not a `package.json` file anymore 😅